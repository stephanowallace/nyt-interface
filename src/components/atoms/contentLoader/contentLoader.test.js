import React from 'react';
import { shallow } from 'enzyme';
import ContentLoader from '.';

describe('Content loader', () => {
  it('Renders default state', () => {
    const component = shallow(<ContentLoader height="150px" width="150px" />);
    expect(component).toMatchSnapshot();
  });
});
