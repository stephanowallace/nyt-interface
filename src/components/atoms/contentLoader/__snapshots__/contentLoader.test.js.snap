// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content loader Renders default state 1`] = `
<div
  className="container "
  style={
    Object {
      "height": "150px",
      "width": "150px",
    }
  }
>
  <div
    className="loader"
  />
</div>
`;
